src/app/tests/suites/certification/Test_TC_DGGEN_2_1.yaml

```diff
@@ -47,7 +47,14 @@ tests:
     - label: "Step 1a: Reset Devices to factory defaults"
       PICS: PICS_USER_PROMPT
       verification: |
-          Reset Devices to factory defaults
+          To perform Factory reset on DUT(all-clusters-app)  follow the below step in RPI platform , Pls use equivalent command on the respective platform
+          sudo rm -rf /tmp/chip_*
+
+          execute the below mentioned command to put DUT into a commissionable state, Pls use equivalent command on the respective DUT
+
+          step-1:  create a file using touch command , something like touch mytest.txt
+          step-2:  chmod 777 mytest.txt
+          step-3:  launch the app  ./chip-all-clusters-app --KVS ./mytest.txt
```

Comment:
But if we are using `./mytest.txt` for KVS, then why are we removing `/tmp/chip_*`?  This makes no sense.

Reply:
Apologies I misunderstood, reverted back these changes as this is not needed for issue. https://github.com/project-chip/connectedhomeip/issues/29331